packages/web/src/styles/response-carousel.css

```
@@ -0,0 +1,63 @@
+    opacity: 0.28;
+}
+
+.response-versions__dots {
+    display: flex;
+    gap: 0.45rem;
+}
+
+.response-versions__dot {
+    width: 0.7rem;
+    height: 0.7rem;
+    border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)